### PR TITLE
Document CMake configure & build wrappers (#158)

### PR DESCRIPTION
**Pull request by sixten-hilborn**
See #158.

### CODE REVIEW

File: conanfile_py.rst [hunk 1]
```diff
@@ -88,13 +90,13 @@ If you are building your project with CMake, edit your ``conanfile.py`` and add
    
       def build(self):
          cmake = CMake(self.settings)
-         self.run('cmake "%s" %s' % (self.conanfile_directory, cmake.command_line))
-         self.run('cmake --build . %s' % cmake.build_config)
+         cmake.configure(self)
```
Comment (lasote) [inline conanfile_py.rst]: At least at the end of this section I would like to mention that `configure` and `build` are only helpers and you can still use the "command line version".

**Comment by lasote** [comment]
Very good job @sixten-hilborn! I did just an observation but it can be merged.

File: conanfile_py.rst (hunk 1 above)
Comment (sixten-hilborn) [inline conanfile_py.rst]: Sure! Agree the "command line version" should be mentioned, it's still the one currently used by most packages.
I can do an update maybe this evening or tomorrow.

**Comment by memsharded** [comment]
The docs are very good, so they can just go live today (I changed the PR to master, develop will not go live until next release). If you can, just add a few lines at the beginning explaining the old command-line mode, before introducing the improved helper. If not, we will add them, dont worry. Thanks very much, great job!

**Pull request merged**